src/features/user/TreeMapper/components/TreeMapperList.tsx

```diff
@@ -2,34 +2,32 @@ import type { ReactElement } from 'react';
 import type { Links } from '../../../common/types/payments';
 import type { SetState } from '../../../common/types/common';
 import type {
-  PlantLocation as PlantLocationType,
-  PlantLocationSingle,
-  PlantLocationMulti,
-} from '../../../common/types/plantLocation';
+  Intervention,
+  InterventionSingle,
+  InterventionMulti,
+} from '../../../common/types/intervention';
 
 import React from 'react';
 import TransactionListLoader from '../../../../../public/assets/images/icons/TransactionListLoader';
 import TransactionsNotFound from '../../../../../public/assets/images/icons/TransactionsNotFound';
 import styles from '../TreeMapper.module.scss';
 import { useTranslations } from 'next-intl';
-import PlantLocation from './PlantLocation';
+import TreemapperIntervention from './TreemapperIntervention';
 
 interface Props {
-  selectedLocation: PlantLocationSingle | PlantLocationMulti | null;
-  setSelectedLocation: SetState<
-    PlantLocationSingle | PlantLocationMulti | null
-  >;
-  plantLocations: PlantLocationType[];
+  selectedLocation: InterventionSingle | InterventionMulti | null;
+  setSelectedLocation: SetState<InterventionSingle | InterventionMulti | null>;
+  intervention: Intervention[];
   isDataLoading: boolean;
-  location: PlantLocationSingle | PlantLocationMulti | null;
+  location: InterventionSingle | InterventionMulti | null;
   fetchTreemapperData: Function;
   links: Links | undefined;
```

Comment:
1. `location`/`selectedLocation` etc is unclear and should be renamed
2. Pluralize `intervention` as it represents an array